README.md

```diff
@@ -34,9 +34,11 @@ over its REST API.
 This version of Hibernate Search requires:
 
 * Java SE 8
-* Hibernate ORM 5.2.x
+* Hibernate ORM 5.2.3.Final or a later 5.2.x
```

Comment:
In my latest release announcement, I mentioned 5.2.7, but indeed I can't find any reason why 5.2.3, 5.2.4, 5.2.5 or 5.2.6 shouldn't work. @Sanne could you confirm this? Maybe I should correct the blog post?